processing/src/main/java/io/druid/query/aggregation/FilteredAggregatorFactory.java

```diff
@@ -28,7 +28,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-public class FilteredAggregatorFactory implements AggregatorFactory
+public class FilteredAggregatorFactory extends AggregatorFactory
```

Comment:
Fwiw this is part of why I was thinking AggregatorFactory still makes sense as an interface… the default impl is not necessarily super universal, it only applies to simple non-configurable aggregators.